SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -127,8 +127,8 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 			&& LocalDeploymentManager->IsLocalDeploymentRunning())
 		{
 			AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this] {
-				const USpatialGDKEditorSettings* Settings = GetDefault<USpatialGDKEditorSettings>();
-				bool bRuntimeShutdown = Settings->bShutdownRuntimeGracefullyOnPIEExit
+				const USpatialGDKEditorSettings* CurSpatialGDKEditorSettings = GetDefault<USpatialGDKEditorSettings>();
```

Comment:
Why does this use CurSpatialGDKEditorSettings instead of SpatialGDKEditorSettings?

Reply:
Because the below code in this function has variable named SpatialGDKEditorSettings, although they are not conflict, I still think the same variable name in the same function will lose code readability.